scripts/ci_checks/linux-checks/check-version.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+
+tagname=$(git tag -l)
+tagname=${tagname: -5}
+header_tagname=$(python3 scripts/scons_helpers/parse-version.py)
+
+if [[ "$tagname" != "$header_tagname" ]]; then
+      echo
+      echo "version check FAILED"
+      echo "the git tag name and version declared in version.h is not the same"
+      echo "${tagname} git version tag"
+      echo "${header_tagname} header version tag"
+      echo
```

Comment:
`echo >&2` is preferred

Reply:
Hey so in the other CI check scripts, such as `check-formatting` , `echo` is used, even for error messages, should I stick with that or make the change to `echo >&2` ?

Reply:
@dheerajd5 , hello, this behavior is preferred, yes, as soon as it doesn't break the script's functionality.